### DISCUSSION TITLE
【点击❤️查看❤️邦布❤️可爱❤️录像】

### DISCUSSION THREAD

**Discussion by TQSJ-TPR** [share121/inter-knot#660] [其他]
![npQ187-3sr3ZbT3cSp2-gp](https://github.com/user-attachments/assets/0e7422c9-e856-45a3-af16-7b232fa1fecc)
<a href="https://www.bilibili.com/video/BV1he4y1w7wB" >点击此处</a>
↑↑↑↑

**Comment by lizard315315** [comment]
❤️物❤️美❤️价❤️廉❤️

**Comment by xiannvanlin** [comment]
你这网页无效啊，我有个朋友想借一部说话

**Comment by BitterWind** [comment]
话说确实随机刷新这类东西来着的,审核机制好像不存在?

**Reply by QingMingqwq** [reply]
绳网目前好像确实没有审核机制，都是管理员主动删的欸..

**Comment by niegong** [comment]
谢谢 孩子很爱吃

**Comment by Summer-rain897** [comment]
这个委托我接了。

**Comment by Lanxey** [comment]
连绳友都骗，别让我在空洞碰到你

**Comment by SDCOM-0415** [comment]
邦布可爱捏
**在空洞遇到你指定有你好果汁吃**

**Comment by Lephaethon** [comment]
你在空洞里小心点

**Comment by xiaomei-aihongwen** [comment]
射出来惹

**Comment by aaa00073** [comment]
我超麦片哥，哪有邦布，借一部说话～

**Comment by AnYuneneu** [comment]
豪赤

**Comment by 2Mnn** [comment]
劳斯，有没有邦布本子推荐？

**Comment by JYZLBL** [comment]
好涩的皮鼓

**Comment by LYBQ** [comment]
不是哥们儿！！！

**Comment by shenguojia123** [comment]
很好看，我已经**社保了！！！** **爽！！！**

**Comment by 3112024746** [comment]
被骗了，但还是想看看这个大邦布的正面

**Comment by dhhdgendjdbvedj** [comment]
空洞里有你好果子吃

**Comment by YoKiiiiiiiiiiiii** [comment]
可恶，❤️我要把你teRiRi掉哦～❤️

**Comment by lllliufeng** [comment]
你最好别进空洞

**Comment by aizai1173** [comment]
太可恶了，必须狠狠的感电

**Comment by L502567** [comment]
呼叫治安官

**Comment by LXH17** [comment]
呼叫朱鸢长官

**Comment by Yith123** [comment]
诈骗!!!

**Comment by nightrain2333** [comment]
我要报警了<img src="https://img-static.mihoyo.com/communityweb/upload/1b5600e9d6d16de86447348909796aa4.png" alt="安比 哭哭" width="75" >
